testing/cert.go

```diff
@@ -63,7 +63,22 @@ func mustParseServerCert(srvCert string, srvKey string) *tls.Certificate {
 }
 
 func mustParseCertAndKey(certPEM, keyPEM string) (*x509.Certificate, *rsa.PrivateKey) {
-	cert, key, err := utilscert.ParseCertAndKey(certPEM, keyPEM)
+	tlsCert, err := tls.X509KeyPair([]byte(certPEM), []byte(keyPEM))
```

Comment:
Nice. Using standard library features directly... how dare you 😆

Reply:
this is because our utils lib now does ed25519 and mongo still uses rsa :-)